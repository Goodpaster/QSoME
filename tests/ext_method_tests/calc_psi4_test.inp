#Geometries taken from

subsystem
H       0.0     0.0     0.0
H       0.0     0.0     1.0
charge 0
hl_method_num 1
end

subsystem
H       0.0     0.0     2.0
H       0.0     0.0     3.0
charge 0
end

basis
 default cc-pvdz
end

env_method_settings
 env_method hf
 embed_settings
  subcycles 50
 end
end


hl_method_settings
    hl_order 1
    hl_method cc3
    use_ext psi4
    excited
    excited_settings
      nroots 3
      cc3_root 2
    end
end
