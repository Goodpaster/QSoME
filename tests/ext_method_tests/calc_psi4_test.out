********************************************************************************
                        ____  _____      ___  ___ _____                         
                       |  _ \/  ___|     |  \/  ||  ___|                        
                       | | | \ `--.  ___ | .  . || |__                          
                       | | | |`--. \/ _ \| |\/| ||  __|                         
                       \ \/' /\__/ / (_) | |  | || |___                         
                        \_/\_\____/ \___/\_|  |_/\____/                         

                  Quantum Solid state and Molecular Embedding                   
                                 Version 0.6.0                                  
                          github.com/Goodpaster/QSoME                           
                          DOI: 10.5281/zenodo.3356913                           
Program Citation: QSoME 0.5, Graham, D.S.; Chulhai, D. V.; Wen, X.; Goodpaster, J. D. University of Minnesota, Minneapolis MN, 2019.

                                    Authors                                     
                                    -------                                     
                      Daniel S. Graham (graha682@umn.edu)                       
                     Dhabih V. Chulhai (chulhaid@uindy.edu)                     
                     Jason D. Goodpaster (jgoodpas@umn.edu)                     

                                   Citations                                    
                                   ---------                                    
               1. J. Chem. Theory Comput. 2017, 13, 1503--1508.                 
               2. J. Chem. Theory Comput. 2018, 14, 1928--1942.                 

********************************************************************************
                                   Input File                                   
********************************************************************************
#Geometries taken from

subsystem
H       0.0     0.0     0.0
H       0.0     0.0     1.0
charge 0
hl_method_num 1
end

subsystem
H       0.0     0.0     2.0
H       0.0     0.0     3.0
charge 0
end

basis
 default cc-pvdz
end

env_method_settings
 env_method hf
 embed_settings
  subcycles 50
 end
end


hl_method_settings
    hl_order 1
    hl_method ccsd
    use_ext psi4
end

********************************************************************************
********************************************************************************
                       Generate Initial System Densities                        
********************************************************************************
********************************************************************************
                            Supersystem Calculation                             
********************************************************************************
converged SCF energy = -2.16649146575005
********************************************************************************
KS-DFT  Energy:                                                      -2.16649147  
TIMING: Supersystem Energy                                               1.0628s
KS-DFT  Energy:                                                      -2.16649147  
TIMING: Supersystem Energy                                               0.0000s
KS-DFT  Energy:                                                      -2.16649147  
TIMING: Supersystem Energy                                               0.0000s
********************************************************************************
TIMING: Initialize Densities                                             2.0080s
********************************************************************************
                                Freeze-and-Thaw                                 
********************************************************************************
iter:  1:0                |ddm|:7.359520e-01               |Tr[DP]|:9.188239e-02
iter:  1:1                |ddm|:7.359520e-01               |Tr[DP]|:9.188239e-02
iter:  2:0                |ddm|:3.985243e-01               |Tr[DP]|:1.619769e-01
iter:  2:1                |ddm|:3.985243e-01               |Tr[DP]|:1.619769e-01
iter:  3:0                |ddm|:3.341959e-02               |Tr[DP]|:1.294714e-01
iter:  3:1                |ddm|:3.341959e-02               |Tr[DP]|:1.294714e-01
iter:  4:0                |ddm|:2.485142e-02               |Tr[DP]|:1.282024e-01
iter:  4:1                |ddm|:2.485142e-02               |Tr[DP]|:1.282024e-01
iter:  5:0                |ddm|:1.032643e-02               |Tr[DP]|:1.279006e-01
iter:  5:1                |ddm|:1.032643e-02               |Tr[DP]|:1.279006e-01
iter:  6:0                |ddm|:1.246415e-03               |Tr[DP]|:1.270983e-01
iter:  6:1                |ddm|:1.246415e-03               |Tr[DP]|:1.270983e-01
iter:  7:0                |ddm|:2.652312e-04               |Tr[DP]|:1.272027e-01
iter:  7:1                |ddm|:2.652312e-04               |Tr[DP]|:1.272027e-01
iter:  8:0                |ddm|:1.040460e-05               |Tr[DP]|:1.271806e-01
iter:  8:1                |ddm|:1.040460e-05               |Tr[DP]|:1.271806e-01
iter:  9:0                |ddm|:2.125821e-07               |Tr[DP]|:1.271813e-01
iter:  9:1                |ddm|:2.125821e-07               |Tr[DP]|:1.271813e-01
iter: 10:0                |ddm|:3.545581e-09               |Tr[DP]|:1.271813e-01
iter: 10:1                |ddm|:3.545577e-09               |Tr[DP]|:1.271813e-01
iter: 11:0                |ddm|:8.316421e-10               |Tr[DP]|:1.271813e-01
iter: 11:1                |ddm|:8.316427e-10               |Tr[DP]|:1.271813e-01
iter: 12:0                |ddm|:2.302122e-10               |Tr[DP]|:1.271813e-01
iter: 12:1                |ddm|:2.302120e-10               |Tr[DP]|:1.271813e-01
                                                                                
Subsystem 0 Energy:                                                   0.84750741
Subsystem 1 Energy:                                                   0.84750741
                                                                                
********************************************************************************
TIMING: Freeze and Thaw                                                  3.0223s
KS-DFT  Energy:                                                      -2.16649147  
TIMING: Supersystem Energy                                               0.0000s
********************************************************************************
                             Env-in-Env Calculation                             
********************************************************************************
Env-in-Env Energy:                                                   -1.79427591
********************************************************************************
TIMING: Env. in Env. Energy                                              0.0004s
********************************************************************************
                           HL Subsystems Calculation                            
********************************************************************************
use external method for hl calculation
Reading embedded core Hamiltonian
Energy:                                                               0.81231298
********************************************************************************
TIMING: Active Energy                                                  102.0446s
********************************************************************************
                           Env Subsystem Calculation                            
********************************************************************************
Energy Subsystem 0:                                                   0.84750741
Energy Subsystem 1:                                                   0.84750741
********************************************************************************
TIMING: Env Energy                                                       0.0178s
********************************************************************************
Total Embedding Energy:     -2.2016858939114994
********************************************************************************
